src/fiatconnect/LinkAccountScreen.tsx

```diff
@@ -120,8 +129,11 @@ FiatConnectLinkAccountScreen.navigationOptions = ({
       }}
     />
   ),
-  // NOTE: title should be dynamic when we support multiple fiat account types
-  headerTitle: i18n.t('fiatConnectLinkAccountScreen.bankAccount.header'),
+  headerTitle: i18n.t(
+    `fiatConnectLinkAccountScreen.${convertAccountTypeToTranslation(
+      route.params.quote.getFiatAccountType()
+    )}.header`
```

Comment:
As a norm we should try to avoid making strings parameterized like this. you can have a similar function, but the strings should be written out in full like `fiatConnectLinkAccountScreen.mobileMoney.header` and `fiatConnectLinkAccountScreen.bankAccount.header`. This makes them easier to find when searching for the code and potentially do in an automated way in the future.

Reply:
Good point, I've updated the code so it keeps the string intact.